luigi/scheduler.py

```diff
@@ -691,6 +717,18 @@ def update_resources(self, **resources):
             self._resources = {}
         self._resources.update(resources)
 
+    def _get_task_level_config(self, config):
```

Comment:
I don't like this function too much. Is `config` a `dict()` type? If so call it `config_dict`, as I thought initially it was a `Configuration` variable (from the standard libraries).